### DISCUSSION TITLE
[React] Create an introductory panel before displaying the StoryList in Map view

### DISCUSSION THREAD

**Discussion by rudokemper** [Terrastories/terrastories#762] [Temporarily Archived Tickets]
Currently, when a user enters the Map, the StoryList in the sidebar is displayed immediately. There is a card on the right side of the screen that pops up and explains a bit about the terrastories app.

Let’s update the UI so that there is no more card and instead, we display an introductory panel in the sidebar before we display the list of Stories.

See below for the design.
![Screen Shot 2020-09-03 at 5 46 06 PM](https://user-images.githubusercontent.com/8590508/92176889-641f9b80-ee0d-11ea-8bfb-afc72bda2b61.png)

note: copy change from this design -- Let's end the explore text at "all over" and not make it specific to the Matawai

We’ll likely need to create a new component which will be rendered into the Sidebar. Then the Sidebar will conditionally display this panel or the list. The condition can be based on some component state which determines whether or not the “Explore” button on this screen has been clicked in this session.

**Acceptance Criteria:**

- [ ] When a user goes to view the Story Map, they should see the introductory panel before they see the Stories.
- [ ] The user should not see the introductory panel after clicking the “Explore” button and should only see Stories.
- [ ] If the user refreshes the page, the introductory panel will be back. 
- [ ] The card on the right of the screen is fully deleted.

**Bonus points:** Add Jest tests

**Comment by griseduardo** [comment]
Hello, I would like to take on this issue.

**Comment by rudokemper** [comment]
Welcome @griseduardo! I will assign you. Please let us know if you have any questions :)

**Comment by griseduardo** [comment]
Hello, just one question about the text.
I updated the i18n en with introductional panel new texts, however the other languages I don't know what to write.
What do I add in other i18n languages?

**Comment by lauramosher** [comment]
@griseduardo English is fine to fill in for languages you don't know.
